_posts/2024-09-24-firo-liquidity-campaign-sept24.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Firo 24-Week Liquidity Mining Campaign"
+summary: "Total reward pool of up to US$19,200 up for grabs"
+tags: [community, news]
+author: "Augustus Jong"
+img: "/blog/assets/firo-liquidity-campaign-sept24/banner.png"
+---
+We are excited to share that the Liquidity campaign is back! Now for 24 weeks with a total reward pool of approximately ~US$19,200 up for grabs! Provide liquidity on Binance and AscendEX via Hummingbot and earn Firo!
+
+![](/blog/assets/firo-liquidity-campaign-sept24/img1.jpg)
+
+## Campaign Terms
+
+* Start date: September 24, 2024, 12:00 AM UTC 
+
+* Total reward pool*: ~US$19,200 (FIRO 684 per week) 
+
+* Reward token: FIRO
+
+* Eligible token pair: FIRO/USDT
+
+* Eligible orders: maker orders placed with spreads of 2.0% or lower
+
+* Exchange: Binance and AscendEX
+
+## What is Hummingbot?
+
+Hummingbot Liquidity Mining is a decentralized, community-based market making. It allows anyone (communities, the general market) to participate in market making for a coin/token and contribute to its liquidity and improve the tradability of that coin/token. By participating, users are not only helping to support the coin/token and the project, but through liquidity mining, they are able to earn coin/token rewards based on their trading activity.
+
+## How Does the Campaign Work?
+
+During the campaign, users who provide liquidity to the FIRO/USDT pair on Binance and AscendEX will earn rewards in Firo weekly.
+
+## How to Get Started?
+
+To join the campaign, you'll need to set up your wallet using the [Hummingbot Miner App](https://miner.hummingbot.io/). A detailed setup guide and How-To Articles are available on the [Hummingbot website](https://support.hummingbot.io/miner/getting-started). After installation, you can follow the instructions to connect the app to the Binance and AscendEX exchanges and begin providing liquidity to the FIRO/USDT pool.
+
+Don't miss this amazing chance to earn rewards while engaging with the Firo community! Whether you're a current participant or new to liquidity mining, we invite you to take part and enjoy the benefits firsthand.
+
+
+*For full terms of the campaign, please read the full terms here [Hummingbot New 24-week for FIRO liquidity mining campaign!](https://support.hummingbot.io/miner/new-24-week-for-firo-liquidity-mining-campaign). Learn more about Liquidity mining [here](https://support.hummingbot.io/miner/liquidity-mining-explained)

```

Comment:
Recommend 'Learn more' on a new line.

For previous link, remove period as there is already a !